src/en/darkscience/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext {
+    extName = 'Dark Science'
+    extClass = '.DarkScience'
+    extVersionCode = 2
```

Comment:
```suggestion
    extVersionCode = 0
```

Reply:
Hey there, question regarding that:

What *is* the state of versions here anyway? The docs seemed to imply that that’s the micro version of the extension API used, which would make it `2` because the linked extension API is `1.4.2`. But you setting that to zero now makes me think this is a mere monotonically incrementing counter for this single extension’s version? Which is it?

Reply:
Yes it is a incrementing version number of the extension, the app prompts for update when this number increases. The major part (1.4) is the major ext-lib version.

And sorry it should be 1 instead of 0. I mistook it for another one when I took a quick look

Reply:
Setting it to zero makes APK compilation fail.